mappings/net/minecraft/client/gui/hud/debug/DebugHud.mapping

```diff
@@ -4,6 +4,7 @@ CLASS net/minecraft/unmapped/C_epxplegp net/minecraft/client/gui/hud/debug/Debug
 	FIELD f_demcwxlr client Lnet/minecraft/unmapped/C_ayfeobid;
 	FIELD f_fjhfdzhz HEIGHT_MAP_TYPES Ljava/util/Map;
 	FIELD f_hdjainem fluidHit Lnet/minecraft/unmapped/C_eetoegzn;
+	FIELD f_jvalscle serverPerformanceSampleLogs Ljava/util/Map;
```

Comment:
While mapping these debug hud related things, I've also got some changes locally to everything related to performance sampling and logging, should I make a separate PR for that, or should I cherry-pick those onto this PR?

Reply:
For example, both GUI code and server-related code reference `net/minecraft/client/gui/hud/debug/PerformanceSampleLog`, so the package is probably wrong. I've also mapped all of the related types and changed some names, which I believe doesn't really fit into the nature of this PR, focusing mostly on adding mappings.

Reply:
I would put those refactors in another PR, this one is fairly huge already